lib/process-services/task-list/components/task-list.component.ts

```diff
@@ -82,23 +83,13 @@ export class TaskListComponent implements OnChanges, AfterContentInit, Paginated
     @Input()
     landingTaskId: string;
 
-    /** Data source object that represents the number and the type of the columns that
-     * you want to show.
-     */
-    @Input()
-    data: DataTableAdapter;
```

Comment:
we can not remove those input you can deprecate it

Reply:
OK

Reply:
1 - mark the property as deprecated (code and documentation)
2 - rollback the hmtl change  
```
<datatable
[data]="data">
```
3 - change the method like
```
ngAfterContentInit() {
// will be removed with the new ADF version
if (this.data === undefined || null) {
    this.createDatatableSchema();
}
```
This means, in case the tasklist provide his data adapter, the datatable will use it.
Otherwise the data table will initialite a new data atapter based on the schema